src/com/lilithsthrone/utils/Colour.java

```diff
@@ -513,62 +517,77 @@
 	// Skin/fur/body part groups:
 	
 	public static List<Colour> humanSkinColours = Util.newArrayListOfValues(
-			Colour.SKIN_PALE,
-			Colour.SKIN_LIGHT,
-			Colour.SKIN_OLIVE,
-			Colour.SKIN_DARK,
-			Colour.SKIN_EBONY);
+
+			new ListValue<Colour>(Colour.SKIN_PORCELAIN),
+			new ListValue<Colour>(Colour.SKIN_PALE),
+			new ListValue<Colour>(Colour.SKIN_LIGHT),
+			new ListValue<Colour>(Colour.SKIN_ROSY),
+			new ListValue<Colour>(Colour.SKIN_TANNED),
+			new ListValue<Colour>(Colour.SKIN_OLIVE),
+			new ListValue<Colour>(Colour.SKIN_CHOCOLATE),
+			new ListValue<Colour>(Colour.SKIN_DARK),
+			new ListValue<Colour>(Colour.SKIN_EBONY));
```

Comment:
It looks like Util.newArrayListOfValues() was adjusted to not require new ListValue<>()'s -- see ratSkinColous. This change should take that into account.